<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>20</y>
      <width>591</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblCam">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btAc">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>610</y>
      <width>591</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Kamera Aç</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btKapat">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>640</y>
      <width>591</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>İptal</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pB">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>670</y>
      <width>581</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QSlider" name="Slider">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>700</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
